action.yml

```diff
@@ -1,12 +1,9 @@
 name: Inject/Extract Cache
 description: "Injects the cached data into the docker build(x|kit) process"
 inputs:
-  cache-source:
-    default: cache
-    description: "Where the cache is stored in the calling workspace. Default: `cache`"
-  cache-target:
-    default: /root/.cache/go-build
-    description: "Where the cache is stored in the docker container. Default: `/root/.cache/go-build`"
```

Comment:
Can we continue to support the old form, with a deprecation notice?

Reply:
Yeah, I can add a compatibility layer.